webapp/client/src/components/StepNavButtons.js

```diff
@@ -1,6 +1,9 @@
 import PropTypes from "prop-types";
 import { useTranslation } from "react-i18next";
+// eslint-disable-next-line
 import styled from "styled-components";
+// eslint-disable-next-line
+import { css } from "styled-components";
```

Comment:
Do you have an idea how I can do this in only one line?

Reply:
I would say:
```suggestion
// eslint-disable-next-line
import styled, { css } from "styled-components";
```

Because styled is the default export and css is explicitly imported (s. [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import)). Or does this not work?

Reply:
Oh nice, I didn't know this was possible. Thanks!